fastlane/Fastfile

```diff
@@ -273,6 +273,10 @@ end
 
 desc "Run automatic bump in other repos"
 lane :bump_hybrid_dependencies do |options|
+  if current_version_prerelease?
+    UI.error("Hybrids won't update automatically for PHC prerelase versions")
```

Comment:
This will make the pipeline fail, but I guess that's ok.

Reply:
Actually, this won't make the pipeline fail. In order to do that, I would need to use `user_error!`. In this case, this is just logging but ends successfully